rego-filters/Trivy AWS Findings

```diff
@@ -0,0 +1,5 @@
+package postee
+
+allow {
+    contains(input.Findings[0].ProductFields["Product Name"], "Trivy")
```

Comment:
About the names inside the rego-filters, don't be better use a pattern with lowercase and without spaces, for example: `trivy-aws-findings`.

Also maybe we could use a pattern to create the sample rego-filters, for example [inputEvent]-[sample].rego. 

Allow-Image-Name.rego -> trivy-allowImage.rego
Tracee Default Set -> tracee-default-set

Reply:
Maybe, today the filenames are doubled up as UI element names. We'll need to change that if we want to define a naming schema.